holmes/plugins/toolsets/elasticsearch/elasticsearch.py

```
@@ -292,181 +121,109 @@ def __init__(self, name: str, description: str, tools: list, **kwargs):
-        except requests.exceptions.ConnectionError as e:
+            return False, f"Elasticsearch SSL error: {error_msg}"
+        except requests.exceptions.ConnectionError:
             return (
                 False,
-                f"[{instance.name}] Failed to connect to {instance.api_url}: {e}",
+                f"Failed to connect to Elasticsearch at {self.elasticsearch_config.api_url}",
             )
         except requests.exceptions.Timeout:
-            return False, f"[{instance.name}] Health check timed out for {instance.api_url}"
+            return False, "Elasticsearch health check timed out"
         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)